docs/source/distributions.rst

```diff
@@ -477,6 +477,13 @@ ZeroInflatedNegativeBinomial2
 
 Directional Distributions
 =========================
+SineBivariateVonMises
```

Comment:
nit: add an empty line above. This should be after `ProjectedNormal`

Reply:
Yes, you are right. I'll fix this.